File: problema08/sol_java/Problema08/src/problema08/Problema08.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package problema08;

import java.util.Locale;
import java.util.Scanner;

/**
 *
 * @author darav
 */
public class Problema08 {

    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        entrada.useLocale(Locale.US);

        int metros;
        int contador = 1;
        double pies=0;
        double pulgadas=0;
        double yardas=0;
        int rep=1;
        String mensaje1 = "Metros\t\tYardas\t\tPulgadas\tPies\n ";

        System.out.println("Escriba una cantidad en metros");
        metros = entrada.nextInt();
        System.out.printf("%s",mensaje1);
        
        while (contador <= metros) {
            pulgadas = rep * 39.37;
            pies = rep * 3.28084;
            yardas = rep * 1.094;
            System.out.printf("%d\t\t%.3f\t\t%.3f\t\t%.3f\n", rep, yardas, pulgadas, pies);
            contador = contador + 1;
            
        
        rep = rep + 1;
        }
        
        
    }

  }

File: problema08/sol_python/Problema08.py
contador = 1
rep = 1
mensaje = "Metros\t\tYardas\t\tPulgadas\tPies\n "

print("Escriba la cantidad de metros a convertir:")
metro = float(input())

print(mensaje)
while contador <= metro:
    pulgadas = rep * 39.37
    yardas = rep * 3.28084
    pies = rep * 1.094
    print(rep,"\t\t","{:.3f}".format(pulgadas), "\t", "{:.3f}".format(yardas), "\t\t", "{:.3f}".format(pies), "\n")
    contador = contador + 1
    rep = rep +1